internal/expr/expr_test.go

```diff
@@ -524,17 +530,17 @@ func (s *ExprSuite) TestPrepareErrors(c *C) {
 		prepareArgs: []any{NoTags{}},
 		err:         `cannot prepare expression: type "NoTags" in "&NoTags.*" does not have any db tags`,
 	}, {
-		query:       "SELECT foo FROM t WHERE x = $S",
+		query:       "SELECT foo FROM t WHERE x = $S.*",
 		prepareArgs: []any{S{}},
 		err:         `cannot prepare expression: cannot use slice type "S" outside of IN clause`,
 	}, {
-		query:       "SELECT col1 AS &S FROM t",
```

Comment:
We should probably still have these tests where we try to use a slice in an output expression (as well as the new ones you've added here)

Reply:
I mean a test for doing `SELECT &S.* FROM t`, which I think is different to the parser test.

Reply:
You are right